src/components/searchInput/searchInput.api.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "SearchInput",
+  "category": "form",
+  "description": "Search input for filtering purpose",
+  "example": "https://github.com/wix-private/wix-react-native-ui-lib/blob/master/example/screens/components/SearchInputScreen.tsx",
+  "extends": ["TextInput"],
+  "extendsLink": "https://reactnative.dev/docs/textinput",
+  "props": [
+    {"name": "onDismiss", "type": "() => void", "description": "callback for dismiss action"},
+    {"name": "onClear", "type": "() => void", "description": "On clear button callback."},
+    {"name": "title", "type": "string", "description": "Title prop."},
+    {"name": "showFilterIcon", "type": "boolean", "description": "Pass true to add filter icon on the right."},
```

Comment:
You left the showFilterIcon prop here...

Reply:
removed, added `customRightElement`